pkg/pillar/cmd/domainmgr/metric.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/flextimer"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/shirou/gopsutil/cpu"
+	"github.com/sirupsen/logrus"
```

Comment:
You shouldn't need this. There is a global variable called log of type *base.LogObject, which if used tags the logs with source=domainmgr etc.